### ISSUE TITLE
Non-negative fluorescence output?

### ISSUE THREAD

**Issue by ziw-liu** [mehta-lab/waveorder#163]
Currently waveorder can output negative fluorescence intensity. This leads to inconveniences in visualization (auto-contrast will boost background noise) and downstream numeric processing. I can think of 2 ways to guarantee a non-negative output:

1. Clip by 0.
2. Shift the histogram to the right.

There could be a flag passed to 'apply inverse' to enable this behavior.

**Comment by talonchandler** [comment]
I think it sense to have clipping as a default-off option `clip=False`. My rough experience is that downstream consumers (people and algorithms) often expect smooth reconstructions on the scale of a few pixels, and clipping like this creates sharp edges that can create problems downstream.   

Shifting the histogram avoids the sharp edges, but I think it will lead to different kinds of confusion---should the maximum value shift based on distant regions in the image?

**Comment by mattersoflight** [comment]
@talonchandler @ziw-liu I agree that clipping should be avoided to avoid texture or high-spatial frequency artifacts. I would add a constant offset, just like we adjust the black level for a camera. It can be a parameter to `apply inverse` and we can default it to a small value that avoids negative output in most cases.
